requirements.txt

```diff
@@ -2,6 +2,7 @@ amightygirl.paapi5-python-sdk==1.0.0
 Babel==2.9.0
 beautifulsoup4==4.9.3
 DBUtils==1.3
+debugpy>=1.2.0
```

Comment:
Oooh, actually I think we do want to keep this in requirements_test.txt. requirements_test.txt is basically any non-production necessary requirements. Let's update the comment in requirements_test.txt to note that it's not just GHA specific.

Reply:
If we do so, we need to separate aways to build our Docker images:
1. A production use Docker build that includes debugpy
2. A non-production use Docker build that does not include debugpy.

Could there be a more streamlined process so to combine these two build types?

Reply:
(I think you have those flipped; production docker build WITHOUT debugpy; and non-prod build WITH debugpy). 

olbase/oldev were meant to be that; but we currently use `oldev` for production (which bring in all the testing stuff). We should at some point transition to using only olbase on production, so we don't have any of that.

Reply:
(That way we'll be able to remove the `build` step during our deploys entirely! Which should speed things up :)

Reply:
Sounds good!